tests/test-module.py

```diff
@@ -57,36 +60,41 @@ def tearDown(self):
         }
         self.s3.Bucket(self.bucket_name).delete_objects(Delete=delete_objects_object)
         self.s3.Bucket(self.bucket_name).delete()
-    # loadAndUpdateRemoteEvent tests
+    # load_and_update_remote_event tests
+
     def test_returns_remote_s3_object(self):
         """ Test remote s3 event is returned when 'replace' key is present """
-        result = self.cumulus_message_adapter.loadAndUpdateRemoteEvent(self.event_with_replace, None)
+        result = self.cumulus_message_adapter.load_and_update_remote_event(
+            self.event_with_replace, None)
         assert result == self.s3_object
 
     def test_returns_event(self):
         """ Test event argument is returned when 'replace' key is not present """
-        result = self.cumulus_message_adapter.loadAndUpdateRemoteEvent(self.event_without_replace, None)
+        result = self.cumulus_message_adapter.load_and_update_remote_event(
+            self.event_without_replace, None)
         assert result == self.event_without_replace
 
-    def test_loadAndUpdateRemoteEvent_handles_cma_parameter(self):
+    def test_load_and_update_remote_event_handles_cma_parameter(self):
         """ Test incoming event with 'cma' parameter is assembled into CMA message """
-        result = self.cumulus_message_adapter.loadAndUpdateRemoteEvent(self.event_with_cma, None)
-        expected = { 'foo': 'bar', 'some': 'object'}
+        result = self.cumulus_message_adapter.load_and_update_remote_event(
+            self.event_with_cma, None)
+        expected = {'foo': 'bar', 'some': 'object'}
         self.assertEqual(expected, result)
 
-
-    def test_loadAndUpdateRemoteEvent_does_not_overwrite_configuration(self):
+    def test_load_and_update_remote_event_does_not_overwrite_configuration(self):
         """ Test incoming event with configuration is not overwritten by key in remote event """
-        result = self.cumulus_message_adapter.loadAndUpdateRemoteEvent(self.config_event_with_replace, None)
+        result = self.cumulus_message_adapter.load_and_update_remote_event(
+            self.config_event_with_replace, None)
         expected = {'task_config': self.config_event_with_replace['cma']['task_config'],
                     'input': u':blue_whale:',
                     'replace': self.config_event_with_replace['cma']['event']['replace']}
         self.assertEqual(expected, result)
 
-    # loadNestedEvent tests
-    def test_returns_loadNestedEvent_local(self):
+    # load_nested_event tests
+    def test_returns_load_nested_event_local(self):
         """
-        Test returns 'config', 'input' and 'messageConfig' in expected format (workflow_config backwards compatible)
+        Test returns 'config', 'input' and 'messageConfig' in expected format (workflow_config
+        backwards compatible)
```

Comment:
Suggest moving the entire parenthetical to one line.

Reply:
That would be a 100 char lint violation

Reply:
f560c5b